### ISSUE TITLE
Include 'Views Rules' module

### ISSUE THREAD

**Issue by ghost** [backdrop-contrib/rules#6]
I'd love to see the [Views Rules](https://www.drupal.org/project/views_rules) module included as a sub-module here, especially as Views is in core now...

**Comment by klonos** [comment]
I agree. All contrib modules should support as much of core as possible.

PS: btw, long time Peter.

**Comment by laryn** [comment]
I'm open to this.
